spec/text-editor-component-spec.coffee

```diff
@@ -68,48 +68,111 @@ describe "TextEditorComponent", ->
       expect(nextAnimationFrame).not.toThrow()
 
   describe "line rendering", ->
-    it "renders the currently-visible lines plus the overdraw margin", ->
-      wrapperNode.style.height = 4.5 * lineHeightInPixels + 'px'
+    expectLineRender = (tileNode, {screenRow, top}) ->
```

Comment:
This helper doesn't represent a clear concept to me, which is why it's tough to name. If I could articulate in a sentence I think it means this:

"Expect this tile node to contain a line node at the following offset."

What about `expectTileContainsRow` and passing the tile and the row as positional args. Then you could follow with `top: 10`. I think that might tell the story a bit better.